devices.js

```diff
@@ -1384,6 +1384,15 @@ const devices = [
             };
         },
     },
+    {
+        zigbeeModel: ['DIYRuZ_magnet'],
+        model: 'DIYRuZ_magnet',
+        vendor: 'DIYRuZ',
+        description: 'DIYRuZ contact sensor',
```

Comment:
Do you have a link on how to build this device?

Reply:
yes. I am forgot... fixed